sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -896,6 +897,42 @@ The following add-on capabilities are available in this SDK:
 
 Note that some add-on capabilities will incur additional charges. See pricing: https://azure.microsoft.com/pricing/details/ai-document-intelligence/.
 
+### Get Raw JSON Result
+Use the `send_request` method to send custom HTTP requests and get raw JSON result from HTTP responses.
+
+<!-- SNIPPET:sample_send_request.send_request -->
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest
+from azure.ai.documentintelligence import DocumentIntelligenceAdministrationClient
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+client = DocumentIntelligenceAdministrationClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+
+with client:
+    # The `send_request` method can send custom HTTP requests that share the client's existing pipeline,
+    # Now let's use the `send_request` method to make a resource details fetching request.
+    # The URL of the request should be absolute, and append the API version used for the request.
+    request = HttpRequest(method="GET", url=f"{endpoint}/documentintelligence/info?api-version=2024-07-31-preview")
+    response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
```

Comment:
Did customers want an example of how to get the raw json with a send_request method or using one of the convenience methods? Like `begin_analyze_document()`?

Reply:
The customer is asking for how to get the raw JSON result via Python API, not specifying in which operation. I will add other samples in this PR for getting raw result via `raw_response_hook` in one of the convenience method.